doc/nrf/templates/library_template_README.rst

```diff
@@ -0,0 +1,143 @@
+.. _library_template:
+
+Template: Library
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   Use this template to create pages that describe the libraries included in |NCS|, following these instructions:
+
+   * Use this first section to give a short description (1 or 2 sentences) of the library, indicating what it does and why it should be used.
+
+   * Give the library a concise name that also corresponds to the folder name.
+
+     If the library targets a specific protocol or device, add it in the title before the library name (for example, "NFC:" or "nRF9160:").
+     Do not include the word "library" in the title.
+     Use the provided stock phrases and includes when possible.
+
+   * Sections with ``*`` are optional and can be left out.
+
+     All the other sections are required for all libraries.
+     Do not add new sections or remove a mandatory section without consulting a tech writer.
+     If any new section is needed, try first to fit it as a subsection of one of the sections listed below.
+     When you keep an optional section, remove the asterisk from the title, and format the length of the related RST heading accordingly.
```

Comment:
I think this could be omitted, as it is somewhat obvious and can be adjusted during the review.

Reply:
I'd disagree that this is obvious if we are trying to make a template for people to follow, that could potentially be applied to external contributions. It is just a reminder after all.